distribution/docs/src/main/resources/_contents/_transformers/tika-input-transformer-contents.adoc

```diff
@@ -0,0 +1,61 @@
+==== Tika Input Transformer
+
+The Tika Input Transformer is the default input transformer responsible for translating Microsoft Word, Microsoft Excel, Microsoft PowerPoint, OpenOffice Writer, and PDF documents into a Catalog Metacard.
```

Comment:
Tika supports a lot more formats than this. Should we include a reference to the Tika documentation?